### DISCUSSION TITLE
Timeseries Example Git Notebooks do not work

### DISCUSSION THREAD

**Discussion by deadmoussa** [pycaret/pycaret#3395] [Q&A]
<img width="1218" alt="Screen Shot 2023-03-13 at 10 10 02 AM" src="https://user-images.githubusercontent.com/53586268/224726531-3ad1f7eb-bcfb-4baf-895a-12f70f521f59.png">

**Comment by ngupta23** [comment]
These should be fixed now. Can you check again? Make sure to clear the cache on the browser since nbviewer can sometimes cache old notebooks.

**Reply by Diogo-loureiro** [reply]
@ngupta23 hi, mine is loading anomaly detection tutorial. I cleared cache.
P.S. I am testing using time series module link: https://pycaret.gitbook.io/docs/get-started/modules

**Reply by ngupta23** [reply]
Let's track the official update here: https://github.com/pycaret/pycaret/issues/3418

In the mean time, you can access all these tutorials from here: https://pycaret.gitbook.io/docs/get-started/tutorials#tutorials-time_series-forecasting
